lib/Target/AVR/AVRInstrInfo.cpp

```diff
@@ -45,6 +45,30 @@ extern bool RA_InSpillerCode;
 AVRInstrInfo::AVRInstrInfo()
     : AVRGenInstrInfo(AVR::ADJCALLSTACKDOWN, AVR::ADJCALLSTACKUP), RI() {}
 
+// FIXME: BEFORE MERGING: Is there a cleaner tablegen method of doing this?
```

Comment:
I wish - the overhead of creating a new TableGen generator is quite large. Would be awesome if TableGen generators could be written as ruby scripts or something.